### PR TITLE
Create AssetItemCreateDialog

### PR DESCRIPTION
**Pull request by anngy12**
new PR for #33

### CODE REVIEW

File: src/TuDa.CIMS.Web/Components/ModalButton.razor
Comment (tom03k) [inline src/TuDa.CIMS.Web/Components/ModalButton.razor:1]: Why do `ModalTarget` and `ButtonLabel` have to be nullable?

File: src/TuDa.CIMS.Web/Components/ChemicalItemForm.razor
Comment (tom03k) [inline src/TuDa.CIMS.Web/Components/ChemicalItemForm.razor:1]: Does the warning "Dereference of a possibly null reference." on line 13 by GitHub show a problem in the code?

**Comment by Sc3l3t0n** [comment]
I would propose that we just merge this and think about it after we finished the `ShoppingCart` @anngy12

**Comment by Sc3l3t0n** [comment]
Maybe we should create this Issue now and link it so we do not forget it

**Comment by anngy12** [comment]
> Maybe we should create this Issue now and link it so we do not forget it

I created a Sub-Issue #86.

**Review by Sc3l3t0n — APPROVED** [review]

**Review by tom03k — APPROVED** [review]

**Pull request merged**